mpifileutils.spec

```
@@ -81,12 +81,20 @@ BuildRequires: libattr-devel
 BuildRequires: libfabric1 >= 1.12.0
 %endif
 
+%if (0%{?suse_version} > 0)
+%global __debug_package 1
+%global _debuginfo_subpackages 0
```

Please set this to 1 as it seems we have changed our policy/desire for wanting debuginfo subpackages recently.

```suggestion
%global _debuginfo_subpackages 1
```